app/assets/stylesheets/functions/_strip-units.scss

```diff
@@ -15,7 +15,7 @@
 @function strip-units($value) {
   @if $output-bourbon-deprecation-warnings == true {
     @warn "[Bourbon] [Deprecation] `strip-units` is deprecated and will be " +
-    "removed in 5.0.0. Use the renamed `strip-unit` function instead.";
+ "removed in version 5 and replaced by `strip-unit` function.";
```

Comment:
Can we indent this line as it was before (to line up with `@warn`?

And swap `version 5` for `5.0.0`, so that we speak in specific version numbers?

Finally, I think we are missing the word `the` before `strip-unit`.